save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestSuitesSourceController.kt

```diff
@@ -503,6 +504,37 @@ class TestSuitesSourceController(
         authentication: Authentication,
     ): Mono<List<String>> = testSuitesSourceService.getOrganizationsWithPublicTestSuiteSources().toMono()
 
+    /**
```

Comment:
Does diktat still add these KDocs? We've added several OpenApi annotations to `ignoreAnnotated` in `diktat.yml`, but probably haven't covered them all..

Reply:
actually, I added it manually. but `ignoreAnnotated` explains why I don't see errors on another methods 😄 
will remove kdoc